source/includes/_actions.md

```diff
@@ -0,0 +1,202 @@
+# Actions
+
+## Get All Actions
+
+```shell
+curl --request GET \
+  --url 'https://public-api.lumiformapp.com/api/v2/actions' \
+  --header 'Authorization: Bearer [your token here]' \
+  --header 'Accept: application/json' \
+  --header 'Content-Type: application/json' 
+```
+
+> The above command returns JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "id": 1,
+      "title": "Check the droid attack on the Wookies",
+      "updated_at": 1593378800,
+      "site": {
+        "id": 1,
+        "title": "Temple of Coruscant"
+      },
+      "resolved_at": 1624914800,
+      "resolved_by": null,
+      "created_at": 1598649200,
+      "assignees": [
+        {
+          "id": 2,
+          "name": "Yoda",
+          "email": "yoda@jediknights.com"
+        }
+      ],
+      "status": "in_progress",
+      "due_at": 1630185200,
```

Comment:
Just a note: If we return data in snake case always, please make sure to also convert keys received from API in later usage from camel to snake. We need to keep this API consistent as it's public, used by our clients, and we need to be professional about it 🙂 
I know you know, but doesn't hurt to mention. 😉